lib/releasy.js

```diff
@@ -23,6 +26,67 @@ module.exports = function(opts) {
   config.npmFolder = opts.npmFolder;
   config.dryRun = opts.dryRun;
   config.quiet = opts.quiet;
+  config.unreleased = '## [Unreleased]'
+  config.changelogPath = 'CHANGELOG.md';
+  config.githubAuth = getGithubToken();
+  config.changelogVersion = `\n\n## [${config.newVersion}] - ${year}-${month}-${day}`;
+  config.githubInfo = getGithubRepo();
+  config.release = {
+    name: config.tagName,
+    tag_name: config.tagName,
+    body: getReleaseNotes() || " ",
+    draft: false,
+    prerelease: false,
+  }
+
+  /**
+     * Get and format GITHUB_API_TOKEN.
+     * @return Github Authorization
+    */
+  function getGithubToken() {
+    if (!process.env.GITHUB_API_TOKEN) {
+      console.log('You must to set GITHUB_API_TOKEN env if you want post the Release Notes of your project.\
```

Comment:
`You must to set` -> `You must set`
`if you want post` -> `if you want to post`